### DISCUSSION TITLE
Sommaire d'achats

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1094] [Idées]
Est-ce que ça sera utile d'avoir un petit sommaire d'achats (e.g. total, et totaux pour des labels ou categories) sur la page du suivi d'achats ?

Dans un deuxième temps on peut ajouter l'option d'afficher le sommaire par an/mois/tout

**Comment by gauthierandre** [comment]
Oui, pour avoir présenter l'outil, les users sont intéressé.e.s par le fait d'avoir une synthèse de ce genre.
